tools/custom-loader.mjs

```diff
@@ -18,7 +18,7 @@ export function resolve(specifier, parent, resolve) {
   if (specifier.includes('-web.js')) {
     specifier = specifier.replace('-web.js', '-node.js');
   }
-  const resolved = new url.URL(specifier, parent);
+  const resolved = new url.URL(specifier, parent || 'file:///');
```

Comment:
When I was doing the Mac hackery with @yuinchien I had to add '/localhost' at one point to progress, but, I was using 'file://' (two slashes, not three), so 'file://localhost/', so I will vote for your solution here (figuring also that you likely actually tested it and progressed farther than I did), but wanted to mention this in any case.

Reply:
Also I was fiddling with the specifier, not the parent...

Reply:
Right, I think `file://localhost/path` and `file:///path` are almost equivalent, the host name should not matter.